### ISSUE TITLE
Building The Pipes of Science by Connecting Ideas

### ISSUE THREAD

**Issue by connerlambden** [ResearchHub/researchhub-web#5]
Instead of focusing on individual papers/posts/hypothesis, what if we designed an abstract "relationship object" to represent and allow discussion about relationships between scientific ideas.

## Potential Use Cases

* Paper X directly contradicts paper Y. People can discuss in a place where all relevant info is easily available.
* Hypothesis X builds on hypothesis Y. Having a graphical relationship helps people build more robust intuitions and navigate in "hypothesis space."
* This is a space for discussion meta-scientific, methodological, and epistemic questions e.g science about science.
* Organizational/pedagogical: a commonly accepted hypothesis X can be shown alongside supporting papers/foundational hypothesis to help people quickly organize/learn mental models of scientific knowledge without having to read 100's of papers.

I'm not sure exactly what this would look like, but welcome to all ideas :)

**Comment by yattias** [comment]
@connerlambden I agree that we need some kind of relationship to tie together different document types: hypotheses, papers, ... so that deeper knowledge can be extracted. Furthermore, connecting discussions together can be valuable too.

- What do you think about connecting documents, comments, etc... together via a tagging system?
For example: imagine some docs were tagged with "Covid", "mRNA" tags then clicking on a tag can show a specific feed with matching posts, papers, etc...
- Also by having tags, we can show recommendations/similar docs on papers, posts, ...

**Comment by connerlambden** [comment]
I think taggings not a bad idea, but it's still at the level of individual articles and I think the search feature is more useful than tags. But yeah, maybe for navigating between related articles it could be useful.
